backend/infrahub/workers/infrahub_async.py

```diff
@@ -148,6 +152,15 @@ async def run(
             identifier=str(flow_run.id),
         )
 
+    def _init_logger(self) -> None:
+        """Initialize loggers to use the API handle provided by Prefect."""
+        api_handler = APILogHandler()
+
+        for logger_name in config.SETTINGS.workflow.extra_loggers + DEFAULT_TASK_LOGGERS:
+            logger = logging.getLogger(logger_name)
+            logger.setLevel(logging.DEBUG)
```

Comment:
Do we really want `logging.DEBUG` here?

Reply:
I added an additional setting to control the log level of the extra logger